src/mixins/AddSendRequestByMessage.ts

```diff
@@ -666,6 +671,305 @@ export function AddSendRequestByMessage<TBase extends ConnectorTUIBaseConstructo
       return requestItem
     }
 
+    private async createFormFieldRequestItem() {
+      const whatFormField = await prompts({
+        message: "What kind of form field should be created?",
+        type: "select",
+        name: "settings",
+        choices: [
+          {
+            title: "Boolean form field",
+            value: "BooleanFormFieldSettings",
+          },
+          {
+            title: "Date form field",
+            value: "DateFormFieldSettings",
+          },
+          {
+            title: "Double form field",
+            value: "DoubleFormFieldSettings",
+          },
+          {
+            title: "Integer form field",
+            value: "IntegerFormFieldSettings",
+          },
+          {
+            title: "Rating form field",
+            value: "RatingFormFieldSettings",
+          },
+          {
+            title: "Selection form field",
+            value: "SelectionFormFieldSettings",
+          },
+          {
+            title: "String form field",
+            value: "StringFormFieldSettings",
+          },
+        ],
+      })
+
+      switch (whatFormField.settings) {
+        case "BooleanFormFieldSettings":
+          return await this.createBooleanFormFieldRequestItem()
+        case "DateFormFieldSettings":
+          return await this.createDateFormFieldRequestItem()
+        case "DoubleFormFieldSettings":
+          return await this.createDoubleFormFieldRequestItem()
+        case "IntegerFormFieldSettings":
+          return await this.createIntegerFormFieldRequestItem()
+        case "RatingFormFieldSettings":
+          return await this.createRatingFormFieldRequestItem()
+        case "SelectionFormFieldSettings":
+          return await this.createSelectionFormFieldRequestItem()
+        case "StringFormFieldSettings":
+          return await this.createStringFormFieldRequestItem()
+        default:
+          return console.log("Invalid form field settings")
+      }
+    }
+
+    private async createBooleanFormFieldRequestItem() {
+      const result = await prompts([
+        {
+          message: "Enter a title for the boolean form field",
+          type: "text",
+          name: "title",
+        },
+      ])
```

Comment:
as title is there for every form field .. shouldn't it be above and then injected in the methods?